ps/GetUmbracoBuildEnv.ps1

```diff
@@ -249,8 +249,8 @@ $ubuild.DefineMethod("GetUmbracoBuildEnv",
       $vsPath = $vsPaths[$vsIx2]
 
       if ($vsVersion.Major -eq 16) {
-        $msBuild = "$vsPath\MSBuild\$($vsVersion.Major).0\Bin"
-        $toolsVersion = "16.0"
+        $msBuild = "$vsPath\MSBuild\Current\Bin"
+        $toolsVersion = "Current"
```

Comment:
and then I am not sure about that one - I think we pass "toolVersion" to MSBuild as an arg and I think it has to be a version number, so it would be ok to change the path but not the version - checking

Reply:
yup we have `/tv:"$($this.BuildEnv.VisualStudio.ToolsVersion)"` in Core's `build.ps1`